src/load-script.test.ts

```diff
@@ -319,17 +319,4 @@ describe("loadCustomScript()", () => {
         );
         expect(PromisePonyfill).toHaveBeenCalledTimes(1);
     });
-
-    test("should throw an error when the Promise implementation is undefined", () => {
```

Comment:
Admittedly, this is a bad test that had to delete the global Promise object. I ended up removing this test since I was unable to get it working with vitest. 

I'm thinking we can actually get rid of this custom error message and just let the browser throw a standard error message in IE11 when `window.Promise` is undefined. https://github.com/paypal/paypal-js/blob/main/src/load-script.ts#L117-L119